lib/externalUpdate.js

```diff
@@ -0,0 +1,51 @@
+import personApi from "./personApi";
+import updateOrgchart from "./updateOrgchart";
+import updateSearch from "./updateSearch";
+
+import { logger } from "./config";
+
+class ExternalUpdate {
+  constructor(cfg) {
+    this.personApi = personApi(cfg);
+    this.orgchartService = updateOrgchart(cfg);
+    this.searchService = updateSearch(cfg);
+  }
+
+  handler() {
+    return (req, res) => {
+      ExternalUpdate.getUserId(req.body)
+        .then(userId => this.getAndDistributeUpdate(userId))
```

Comment:
Out of curiosity: Why no async/await?

Reply:
I think this is a nice chain and `Promise.then` looks nicer here.